Docker/nginx/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:jessie
```

Comment:
No longer maintened, and build was not working anyway.

Now build passes, but we get a 403 when trying to reach http://codelabs.local/